### ISSUE TITLE
Request: larger values?

### ISSUE THREAD

**Issue by Arroyodorado** [ozzik/push-and-shove#2]
Hi ozzik, how about making larger distances/amounts possible? For example move or resize by 100px. Of courxe I know, there are limited key combinations to trigger such commands but if you included it in the menu, one could asign his own shortcuts (with Sketch Monster) and do without other commands. What do you think?

**Comment by ozzik** [comment]
Hey, thanks for suggesting this!

So just wondering, since upcoming Sketch 45 will have support for setting how many px you'd like objects to be moved when using shift, do you think something like this would be usable?

Thanks

**Comment by Arroyodorado** [comment]
Definitely, because sometimes you need movements by 100, sometimes by ten and other times by half a pixel. And you don't want to fiddle with settings all the time.
However, I have to admit that I'm currently not using the plugin...
